examples/startAndConnect.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var connect = require('../');
+var startSimulator = require('node-firefox-start-simulator');
+
+startSimulator({}).then(function(simulator) {
```

Comment:
Is the empty options hash required? I don't think it needs to be in the example here :-)